R/broom_wrapper.R

```diff
@@ -148,7 +148,9 @@ add_prediction <- function(df, model_df, conf_int = 0.95, ...){
   # if type.predict argument is not indicated in this function
   # and models have $family$linkinv (basically, glm models have it),
   # both fitted link value column and response value column should appear in the result
-  with_response <- !("type.predict" %in% names(cll)) & !is.null(model_df[["model"]][[1]]$family) & !is.null(model_df[["model"]][[1]]$family$linkinv)
+    with_response <- !("type.predict" %in% names(cll)) &&
```

Comment:
Indent is off here.

Reply:
What is the reason now you are looking at df instead of model_df here?

Reply:
Also would it be same to check s$family$linkinv in a different any(lapply) than the one checking s$family? Wouldn't it possible to end up trying to acces s$family$linkinv when s$family is NULL?

Reply:
Hironori to check.